application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -20,21 +20,21 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 -->
 
-<xwikidoc version="1.1">
+<xwikidoc version="1.3" reference="XPoll.XPollSheet" locale="">
   <web>XPoll</web>
   <name>XPollSheet</name>
   <language/>
   <defaultLanguage/>
   <translation>0</translation>
   <creator>xwiki:XWiki.Admin</creator>
+  <creationDate>1325372400000</creationDate>
   <parent>XPoll.XPollClass</parent>
   <author>xwiki:XWiki.Admin</author>
   <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
-  <creationDate>1325372400000</creationDate>
-  <date>1325372400000</date>
-  <contentUpdateDate>1325372400000</contentUpdateDate>
+  <date>1530721393000</date>
+  <contentUpdateDate>1530721339000</contentUpdateDate>
   <version>1.1</version>
-  <title>#if("$!doc.getObject('XPoll.XPollClass').getProperty('name').value" != '')$!doc.getObject('XPoll.XPollClass').getProperty('name').value #else ${doc.name} #end</title>
+  <title>#if("$!doc.getObject('XPoll.XPollClass').getProperty('name').value" != '')$!doc.getObject('XPoll.XPollClass').getProperty('name').value #elseif($doc.name=='XPollSheet' )${doc.name} #else${doc.title} #end</title>
```

Comment:
There's no need for this change in the title. The `#else` branch will never be reached because the first condition will always be met for the application's entry (when the value is set in `id="XPoll.XPollClass_${pollObj.number}_name" type="text" value="$escapetool.xml($doc.title)"`) and for the `XPollSheet` and `XPollTemaplate` the `$doc.name` is always available.